redhat-developer/developers.redhat.com
# DEVELOPER-4814 & DEVELOPER-4816

All back-end code done to implement the Expandable Product Group assembly.

JIRA: https://issues.jboss.org/browse/DEVELOPER-4816 and https://issues.jboss.org/browse/DEVELOPER-4814

Testing: 

* Modify the landing page section to accept the Expandable Product Group assembly.
* Create an instance of the new assembly
* Create an instance of Landing Page using the assembly created in the previous step
* View the source for the created page
* Verify it matches this layout for the assembly:

```
dp-category-list
  dp-category(name, image)
    dp-category-item-list
      dp-category-item
        dp-product-short-teaser(name, link, download link)
          Product Short Description Text
```
